distribution/openrct2.d.ts

```diff
@@ -854,10 +894,15 @@ declare global {
         speed: number;
     }
 
-    // recommendation: use Peep.setFlag instead of the GuestSetFlag action
+    /**
+     * @todo Recommendation: Use Peep.setFlag instead of the GuestSetFlag action
```

Comment:
This recommendation is outdated. There are situations where game actions are more favourable, for example in [multiplayer](https://github.com/OpenRCT2/OpenRCT2/blob/develop/distribution/scripting.md#multiplayer). I'd suggest removing it.